mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -4867,7 +5330,7 @@
 													"        let firstRecord = response.data[0];",
 													"        ",
 													"        //Test that type is providers",
-													"        pm.test('type is packages', function(){",
+													"        pm.test('type is provider', function(){",
```

Comment:
This is a bit confusing -> are we checking for providers or packages here? comment says provider but we are checking that type should be packages?

Reply:
I fixed it in previous commit. I changed this line to "provider".

Reply:
Still confused @stanislav-mureiko : on line 5333 -> pm.test('type is provider',) but on line 5334 -> pm.expect(firstRecord.type).eq('packages') we expect type to be packages? Sorry, I might be missing something here and thanks in advance for the clarification.